frontend/packages/operator-lifecycle-manager/src/components/operator-install-page.tsx

```diff
@@ -206,41 +216,73 @@ const InitializationResourceRequiredMessage: React.FC<InitializationResourceRequ
 };
 
+const InitializationLinkButton: React.FC<{ disabled?: boolean }> = ({ disabled }) => {
+  const { t } = useTranslation();
+  return (
+    <Button aria-disabled={disabled} isDisabled={disabled} variant="primary">
+      {t('olm~Configure Operator')}
+    </Button>
+  );
+};
+const InitializationLink: React.FC<InitializationLinkProps> = ({ to, disabled }) => {
+  if (!to) {
+    return null;
+  }
+
+  if (disabled) {
+    return <InitializationLinkButton disabled />;
+  }
+
+  return (
+    <Link to={to}>
+      <InitializationLinkButton />
+    </Link>
+  );
+};
+
 const InstallSucceededMessage: React.FC<InstallSuccededMessageProps> = ({
   namespace,
   csvName,
   obj,
 }) => {
   const { t } = useTranslation();
-  const initializationResource = getInitializationResource(obj?.metadata?.annotations, {
+  const annotationParserOptions = {
     onError: (error) => errorModal({ error }),
-  });
+  };
+  const initializationLink = getInitializationLink(
+    obj?.metadata?.annotations,
+    annotationParserOptions,
+  );
+  const initializationResource =
+    !initializationLink &&
+    getInitializationResource(obj?.metadata?.annotations, annotationParserOptions);
+
   return (
     <>
       <Title headingLevel="h2" className="co-clusterserviceversion-install__heading">
-        {t('olm~Installed operator')}: &nbsp;
-        {initializationResource ? t('olm~custom resource required') : t('olm~ready for use')}
+        {!initializationLink && !initializationResource && t('olm~Operator installed successfully')}
+        {initializationLink && t('olm~Configure Operator')}
+        {initializationResource && t('olm~ Create initialization resource')}
       </Title>
-      {initializationResource && (
-        <>
-          <span>
-            {t(
-              'olm~The Operator has installed successfully. Create the required custom resource to be able to use this Operator.',
-            )}
-          </span>
-          <InitializationResourceRequiredMessage
-            initializationResource={initializationResource}
-            obj={obj}
-          />
-        </>
-      )}
+      <span>
+        {t('olm~The Operator has installed successfully.')}
+        {!initializationLink && !initializationResource && t('olm~ Ready for use.')}
+        {initializationLink &&
+          t('olm~ Complete the next configuration steps to prepare it for use.')}
+        {initializationResource &&
+          t('olm~ Create the required custom resource to prepare it for use.')}
+        <InitializationResourceRequiredMessage
+          initializationResource={initializationResource}
+          obj={obj}
+        />
+      </span>
       <ActionGroup className="pf-v6-c-form pf-v6-c-form__group--no-top-margin">
-        {initializationResource ? (
-          <CreateInitializationResourceButton
-            initializationResource={initializationResource}
-            obj={obj}
-          />
-        ) : (
+        <InitializationLink to={initializationLink} />
```

Comment:
Passing raw URL object doesn't work. You need to convert this to string. Perhaps at an earlier place.